### PR TITLE
Add launcher icons for mainnet staging

### PR DESCRIPTION
**Pull request by elgatovital**
@gnosis/mobile-devs

### CODE REVIEW

**Review by biafra23 — APPROVED** [review]
LGTM 👍

File: app/src/mainnet_staging/res/values/static_strings.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name" translatable="false">Safe Multisig - Mainnet Staging</string>
```
Comment (biafra23) [inline app/src/mainnet_staging/res/values/static_strings.xml:3]: Do you see the end of this name in your home screen?

Maybe we could rename it to Mainnet Staging - Safe Multisig?

Comment (elgatovital) [reply app/src/mainnet_staging/res/values/static_strings.xml:3]: or just Mainnet Staging. It is clear anyway that the app is Safe Multisig

Comment (elgatovital) [reply app/src/mainnet_staging/res/values/static_strings.xml:3]: but this name is mainly for app distribution screen. App is identifiable by the icon and nobody should use it, but us. So I think it's fine to just leave it as is and continue using our established app naming schema :)

Comment (elgatovital) [reply app/src/mainnet_staging/res/values/static_strings.xml:3]: So you vote for Mainnet Staging?

**Pull request merged**